libs/dh/wholesale/feature-request-calculation/src/lib/dh-wholesale-request-calculation.ts

```diff
@@ -138,6 +127,9 @@ export class DhWholesaleRequestCalculationComponent {
       Validators.required,
       WattRangeValidators.required(),
```

Comment:
issue: somehow a bug sneaked in the datepicker that displays the required error message even after valid values are (manually) inputted. Doesn't look like the bug was introduced in this PR, so we should address it separately.

![issue](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/8073850c-3027-48a4-b8e6-b63714a1f1ca)

Reply:
I noticed it as well